devenv/config.ini

```diff
@@ -1,5 +1,5 @@
 [devenv]
-minimum_version = 1.10.2
+minimum_version = 1.11.0
```

Comment:
this is a noop until minimum_version is implemented in 1.12.0 (https://github.com/getsentry/devenv/pull/163)

but... 1.11.0 _is_ a hard minimum requirement otherwise the newer colima won't be installed as a shim, and the newer limactl from homebrew can't be installed either

think we do need something in sync.py in the interim to lay the chicken egg